to.etc.domui/src/main/java/to/etc/domui/component/upload/FileUpload2.java

```diff
@@ -313,6 +314,7 @@ private void clearClicked() throws Exception {
 	public void clear() throws Exception {
 		if(m_value == null)
 			return;
+		FileTool.closeAll(m_value.getFile());
```

Comment:
since it might be unclear why is that needed, we could add an comment next to it:
//removes created temp file